blockly-rtc/src/WorkspaceClient.js

```diff
@@ -126,26 +169,75 @@ export default class WorkspaceClient {
     if (!success) {
       this.notSent = this.inProgress[0].events.concat(this.notSent);
       this.inProgress = [];
-      this.counter -=1;
+      this.counter -= 1;
     };
     this.writeInProgress = false;
   };
 
+  /**
+   * Periodically query the database for new server events and add them to
+   * serverEvents.
+   * @private
+   */
+  async pollServer_() {
+    const rows = await this.queryDatabase_();
+    await this.addServerEvents_(rows);
+    this.pollServer_();
```

Comment:
This will run more or less continuously.  What's the plan for knocking it down to a lower frequency?

Reply:
For the time being I will add a timeout to poll every couple of seconds.
The long term solution is to move from continuous polling to long polling.